### ISSUE TITLE
[Bug] Fresh Ubuntu 20.04 Install - Unable to start TheHive4 

### ISSUE THREAD

**Issue by aim4r** [TheHive-Project/TheHive#1464]
### Request Type
Bug
### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Ubuntu 20.04
| TheHive version / git hash   | 4
| Package Type              | DEB

### Problem Description
Unable to start TheHive4 on a fresh Ubuntu 20.04 install

### Steps to Reproduce
Followed these installation steps : https://github.com/TheHive-Project/TheHiveDocs/blob/master/TheHive4/Installation/Install_deb.md
Opted for local filesystem option.

### Complementary information
Application.conf:
```
include "/etc/thehive/secret.conf"

db {
  provider: janusgraph
  janusgraph {
   storage {
    backend: cql
    hostname: ["192.168.1.92"]
    cql {
      cluster-name: thp
      keyspace: thehive
      read-consistency-level: ONE
      write-consistency-level: ONE
    }
   }
  }

}

storage {
  provider: localfs
  localfs.directory: /opt/thp_data/files/thehive
}
```

Error message :
```

cat /var/log/thehive/application.log
2020-08-05 18:28:28,846 [INFO] from org.thp.scalligraph.ScalligraphModule in main [|] Loading scalligraph module
2020-08-05 18:28:31,559 [INFO] from akka.event.slf4j.Slf4jLogger in application-akka.actor.default-dispatcher-6 [|] Slf4jLogger started
2020-08-05 18:28:32,155 [INFO] from akka.remote.artery.tcp.ArteryTcpTransport in application-akka.actor.default-dispatcher-6 [|] Remoting started with transport [Artery tcp]; listening on address [akka://application@127.0.0.1:38925] with UID [-6118572723792472096]
2020-08-05 18:28:32,198 [INFO] from akka.cluster.Cluster in application-akka.actor.default-dispatcher-6 [|] Cluster Node [akka://application@127.0.0.1:38925] - Starting up, Akka version [2.6.5] ...
2020-08-05 18:28:32,431 [INFO] from akka.cluster.Cluster in application-akka.actor.default-dispatcher-6 [|] Cluster Node [akka://application@127.0.0.1:38925] - Registered cluster JMX MBean [akka:type=Cluster]
2020-08-05 18:28:32,432 [INFO] from akka.cluster.Cluster in application-akka.actor.default-dispatcher-6 [|] Cluster Node [akka://application@127.0.0.1:38925] - Started up successfully
2020-08-05 18:28:32,487 [INFO] from akka.cluster.Cluster in application-akka.actor.default-dispatcher-6 [|] Cluster Node [akka://application@127.0.0.1:38925] - No downing-provider-class configured, manual cluster downing required, see https://doc.akka.io/docs/akka/current/typed/cluster.html#downing
2020-08-05 18:28:32,488 [INFO] from akka.cluster.Cluster in application-akka.actor.default-dispatcher-6 [|] Cluster Node [akka://application@127.0.0.1:38925] - No seed-nodes configured, manual cluster join required, see https://doc.akka.io/docs/akka/current/typed/cluster.html#joining
2020-08-05 18:28:34,501 [ERROR] from akka.actor.OneForOneStrategy in application-akka.actor.default-dispatcher-6 [|] Unable to provision, see the following errors:

1) Error injecting constructor, java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
  at org.thp.scalligraph.janus.JanusDatabase.<init>(JanusDatabase.scala:71)
  at org.thp.scalligraph.janus.JanusDatabase.class(JanusDatabase.scala:56)
  while locating org.thp.scalligraph.janus.JanusDatabase
  while locating org.thp.scalligraph.models.Database
    for the 1st parameter of org.thp.thehive.models.DatabaseProvider.<init>(SchemaUpdaterActor.scala:20)
  at org.thp.thehive.models.DatabaseProvider.class(SchemaUpdaterActor.scala:19)
  while locating org.thp.thehive.models.DatabaseProvider
  while locating org.thp.scalligraph.models.Database annotated with @com.google.inject.name.Named(value=with-thehive-schema)
    for the 4th parameter of org.thp.thehive.services.AuditSrv.<init>(AuditSrv.scala:28)
  at org.thp.thehive.services.AuditSrv.class(AuditSrv.scala:27)
  while locating org.thp.thehive.services.AuditSrv
    for the 3rd parameter of org.thp.thehive.services.notification.NotificationActor.<init>(NotificationActor.scala:76)
  while locating org.thp.thehive.services.notification.NotificationActor

1 error
akka.actor.ActorInitializationException: akka://application/user/notification-actor: exception during creation
	at akka.actor.ActorInitializationException$.apply(Actor.scala:196)
	at akka.actor.ActorCell.create(ActorCell.scala:659)
	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:511)
	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:533)
	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:295)
	at akka.dispatch.Mailbox.run(Mailbox.scala:230)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting constructor, java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
  at org.thp.scalligraph.janus.JanusDatabase.<init>(JanusDatabase.scala:71)
  at org.thp.scalligraph.janus.JanusDatabase.class(JanusDatabase.scala:56)
  while locating org.thp.scalligraph.janus.JanusDatabase
  while locating org.thp.scalligraph.models.Database
    for the 1st parameter of org.thp.thehive.models.DatabaseProvider.<init>(SchemaUpdaterActor.scala:20)
  at org.thp.thehive.models.DatabaseProvider.class(SchemaUpdaterActor.scala:19)
  while locating org.thp.thehive.models.DatabaseProvider
  while locating org.thp.scalligraph.models.Database annotated with @com.google.inject.name.Named(value=with-thehive-schema)
    for the 4th parameter of org.thp.thehive.services.AuditSrv.<init>(AuditSrv.scala:28)
  at org.thp.thehive.services.AuditSrv.class(AuditSrv.scala:27)
  while locating org.thp.thehive.services.AuditSrv
    for the 3rd parameter of org.thp.thehive.services.notification.NotificationActor.<init>(NotificationActor.scala:76)
  while locating org.thp.thehive.services.notification.NotificationActor

1 error
	at com.google.inject.internal.InternalProvisionException.toProvisionException(InternalProvisionException.java:226)
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1097)
	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1131)
	at play.api.inject.guice.GuiceInjector.instanceOf(GuiceInjectorBuilder.scala:436)
	at play.api.inject.guice.GuiceInjector.instanceOf(GuiceInjectorBuilder.scala:431)
	at play.api.inject.ContextClassLoaderInjector.$anonfun$instanceOf$2(Injector.scala:119)
	at play.api.inject.ContextClassLoaderInjector.withContext(Injector.scala:128)
	at play.api.inject.ContextClassLoaderInjector.instanceOf(Injector.scala:119)
	at play.api.libs.concurrent.ActorRefProvider.$anonfun$get$1(Akka.scala:268)
	at akka.actor.TypedCreatorFunctionConsumer.produce(IndirectActorProducer.scala:91)
	at akka.actor.Props.newActor(Props.scala:226)
	at akka.actor.ActorCell.newActor(ActorCell.scala:611)
	at akka.actor.ActorCell.create(ActorCell.scala:637)
	... 9 common frames omitted
Caused by: java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
	at org.janusgraph.util.system.ConfigurationUtil.instantiate(ConfigurationUtil.java:60)
	at org.janusgraph.diskstorage.Backend.getImplementationClass(Backend.java:440)
	at org.janusgraph.diskstorage.Backend.getStorageManager(Backend.java:411)
	at org.janusgraph.graphdb.configuration.builder.GraphDatabaseConfigurationBuilder.build(GraphDatabaseConfigurationBuilder.java:50)
	at org.janusgraph.core.JanusGraphFactory.open(JanusGraphFactory.java:161)
	at org.janusgraph.core.JanusGraphFactory.open(JanusGraphFactory.java:132)
	at org.janusgraph.core.JanusGraphFactory.open(JanusGraphFactory.java:112)
	at org.thp.scalligraph.janus.JanusDatabase$.openDatabase(JanusDatabase.scala:51)
	at org.thp.scalligraph.janus.JanusDatabase.<init>(JanusDatabase.scala:72)
	at org.thp.scalligraph.janus.JanusDatabase$$FastClassByGuice$$113881e3.newInstance(<generated>)
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:114)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:168)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:39)
	at com.google.inject.internal.FactoryProxy.get(FactoryProxy.java:62)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:168)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:39)
	at com.google.inject.internal.BoundProviderFactory.get(BoundProviderFactory.java:60)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:168)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:39)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1094)
	... 20 common frames omitted
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.janusgraph.util.system.ConfigurationUtil.instantiate(ConfigurationUtil.java:56)
	... 60 common frames omitted
2020-08-05 18:28:34,877 [INFO] from org.reflections.Reflections in main [|] Reflections took 412 ms to scan 1 urls, producing 142 keys and 2211 values 
2020-08-05 18:28:34,941 [INFO] from org.thp.thehive.ClusterSetup in main [|] Initialising cluster
2020-08-05 18:28:34,983 [INFO] from akka.cluster.Cluster in application-akka.actor.default-dispatcher-11 [|] Cluster Node [akka://application@127.0.0.1:38925] - Node [akka://application@127.0.0.1:38925] is JOINING itself (with roles [dc-default]) and forming new cluster
2020-08-05 18:28:34,989 [INFO] from akka.cluster.Cluster in application-akka.actor.default-dispatcher-11 [|] Cluster Node [akka://application@127.0.0.1:38925] - is the new leader among reachable nodes (more leaders may exist)
2020-08-05 18:28:35,012 [INFO] from akka.cluster.Cluster in application-akka.actor.default-dispatcher-11 [|] Cluster Node [akka://application@127.0.0.1:38925] - Leader is moving node [akka://application@127.0.0.1:38925] to [Up]
2020-08-05 18:28:35,295 [INFO] from akka.cluster.singleton.ClusterSingletonManager in application-akka.actor.default-dispatcher-11 [|] Singleton manager starting singleton actor [akka://application/user/integrityCheckSingletonManager/singleton]
2020-08-05 18:28:35,320 [INFO] from akka.cluster.singleton.ClusterSingletonManager in application-akka.actor.default-dispatcher-6 [|] ClusterSingletonManager state change [Start -> Oldest]
2020-08-05 18:28:35,893 [ERROR] from akka.actor.OneForOneStrategy in application-akka.actor.default-dispatcher-11 [|] Unable to provision, see the following errors:

1) Error injecting constructor, java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
  at org.thp.scalligraph.janus.JanusDatabase.<init>(JanusDatabase.scala:71)
  at org.thp.scalligraph.janus.JanusDatabase.class(JanusDatabase.scala:56)
  while locating org.thp.scalligraph.janus.JanusDatabase
  while locating org.thp.scalligraph.models.Database
    for the 1st parameter of org.thp.thehive.models.DatabaseProvider.<init>(SchemaUpdaterActor.scala:20)
  at org.thp.thehive.models.DatabaseProvider.class(SchemaUpdaterActor.scala:19)
  while locating org.thp.thehive.models.DatabaseProvider
  while locating org.thp.scalligraph.models.Database annotated with @com.google.inject.name.Named(value=with-thehive-schema)
    for the 1st parameter of org.thp.thehive.services.ProfileIntegrityCheckOps.<init>(ProfileSrv.scala:88)
  while locating org.thp.thehive.services.ProfileIntegrityCheckOps
  while locating org.thp.scalligraph.services.GenIntegrityCheckOps annotated with @com.google.inject.internal.Element(setName=,uniqueId=26, type=MULTIBINDER, keyType=)
  while locating java.util.Set<org.thp.scalligraph.services.GenIntegrityCheckOps>

1 error
akka.actor.ActorInitializationException: akka://application/user/integrityCheckSingletonManager/singleton: exception during creation
	at akka.actor.ActorInitializationException$.apply(Actor.scala:196)
	at akka.actor.ActorCell.create(ActorCell.scala:659)
	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:511)
	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:533)
	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:295)
	at akka.dispatch.Mailbox.run(Mailbox.scala:230)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting constructor, java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
  at org.thp.scalligraph.janus.JanusDatabase.<init>(JanusDatabase.scala:71)
  at org.thp.scalligraph.janus.JanusDatabase.class(JanusDatabase.scala:56)
  while locating org.thp.scalligraph.janus.JanusDatabase
  while locating org.thp.scalligraph.models.Database
    for the 1st parameter of org.thp.thehive.models.DatabaseProvider.<init>(SchemaUpdaterActor.scala:20)
  at org.thp.thehive.models.DatabaseProvider.class(SchemaUpdaterActor.scala:19)
  while locating org.thp.thehive.models.DatabaseProvider
  while locating org.thp.scalligraph.models.Database annotated with @com.google.inject.name.Named(value=with-thehive-schema)
    for the 1st parameter of org.thp.thehive.services.ProfileIntegrityCheckOps.<init>(ProfileSrv.scala:88)
  while locating org.thp.thehive.services.ProfileIntegrityCheckOps
  while locating org.thp.scalligraph.services.GenIntegrityCheckOps annotated with @com.google.inject.internal.Element(setName=,uniqueId=26, type=MULTIBINDER, keyType=)
  while locating java.util.Set<org.thp.scalligraph.services.GenIntegrityCheckOps>

1 error
	at com.google.inject.internal.InternalProvisionException.toProvisionException(InternalProvisionException.java:226)
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1097)
	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1126)
	at org.thp.thehive.services.IntegrityCheckActor.integrityCheckOps$lzycompute(IntegrityCheckActor.scala:34)
	at org.thp.thehive.services.IntegrityCheckActor.integrityCheckOps(IntegrityCheckActor.scala:33)
	at org.thp.thehive.services.IntegrityCheckActor.preStart(IntegrityCheckActor.scala:55)
	at akka.actor.Actor.aroundPreStart(Actor.scala:546)
	at akka.actor.Actor.aroundPreStart$(Actor.scala:546)
	at org.thp.thehive.services.IntegrityCheckActor.aroundPreStart(IntegrityCheckActor.scala:25)
	at akka.actor.ActorCell.create(ActorCell.scala:639)
	... 9 common frames omitted
Caused by: java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
	at org.janusgraph.util.system.ConfigurationUtil.instantiate(ConfigurationUtil.java:60)
	at org.janusgraph.diskstorage.Backend.getImplementationClass(Backend.java:440)
	at org.janusgraph.diskstorage.Backend.getStorageManager(Backend.java:411)
	at org.janusgraph.graphdb.configuration.builder.GraphDatabaseConfigurationBuilder.build(GraphDatabaseConfigurationBuilder.java:50)
	at org.janusgraph.core.JanusGraphFactory.open(JanusGraphFactory.java:161)
	at org.janusgraph.core.JanusGraphFactory.open(JanusGraphFactory.java:132)
	at org.janusgraph.core.JanusGraphFactory.open(JanusGraphFactory.java:112)
	at org.thp.scalligraph.janus.JanusDatabase$.openDatabase(JanusDatabase.scala:51)
	at org.thp.scalligraph.janus.JanusDatabase.<init>(JanusDatabase.scala:72)
	at org.thp.scalligraph.janus.JanusDatabase$$FastClassByGuice$$113881e3.newInstance(<generated>)
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:114)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:168)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:39)
	at com.google.inject.internal.FactoryProxy.get(FactoryProxy.java:62)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:168)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:39)
	at com.google.inject.internal.BoundProviderFactory.get(BoundProviderFactory.java:60)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.FactoryProxy.get(FactoryProxy.java:62)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.RealMultibinder$RealMultibinderProvider.doProvision(RealMultibinder.java:198)
	at com.google.inject.internal.RealMultibinder$RealMultibinderProvider.doProvision(RealMultibinder.java:151)
	at com.google.inject.internal.InternalProviderInstanceBindingImpl$Factory.get(InternalProviderInstanceBindingImpl.java:113)
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1094)
	... 17 common frames omitted
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.janusgraph.util.system.ConfigurationUtil.instantiate(ConfigurationUtil.java:56)
	... 54 common frames omitted
2020-08-05 18:28:35,898 [INFO] from akka.cluster.singleton.ClusterSingletonManager in application-akka.actor.default-dispatcher-11 [|] Singleton actor [akka://application/user/integrityCheckSingletonManager/singleton] was terminated

```

**Comment by lybinhlap** [comment]
I had the same issue. Any update from this issue?

**Comment by viszsec** [comment]
same case for me

**Comment by torrycrass** [comment]
This appeared on my v4 install after adding the following configurations to my nginx setup

```
			proxy_connect_timeout   600;
			proxy_send_timeout      600;
			proxy_read_timeout      600;
			send_timeout            600;
			client_max_body_size    2G;
			proxy_buffering off;
			client_header_buffer_size 8k;
```
and
```
proxy_set_header Connection ""; # cf. https://github.com/akka/akka/issues/19542
```

**Comment by viszsec** [comment]
It worked for me with thp4 running 20.04. With the .DEB installation

**Comment by lybinhlap** [comment]
> It worked for me with thp4 running 20.04. With the .DEB installation

Hi @viszsec , please share with me what have done to fix this issue?

**Comment by viszsec** [comment]
You can try a fresh install and follow exactly the installation from https://blog.agood.cloud

**Comment by aim4r** [comment]
This seems resolved in 4.0.2. Closing. Please refer to the install steps over at :
https://github.com/TheHive-Project/TheHiveDocs/blob/master/TheHive4/Installation/Install_deb.md
